pyproject.toml

```diff
@@ -1,65 +1,64 @@
-[project]
-name = "skycalc_ipy"
+[tool.poetry]
+name = "skycalc-ipy"
 version = "0.3.0a0"
-# When updating the version, also
-# - update the date in anisocado/version.py
-# - update the release notese in docs/source/index.rst
-description = "Get atmospheric spectral information from the ESO skycalc server"
-readme = "README.md"
-requires-python = ">=3.8"
-# When updating the version, also update the versions in .github/workflows/*
-license = {text = "License :: OSI Approved :: GNU General Public License v3 (GPLv3)"}
-authors = [
-    {name = "Kieran Leschinski", email="kieran.leschinski@unive.ac.at"},
-]
+description = "Get atmospheric spectral information from the ESO skycalc server."
+license = "GPL-3.0-or-later"
+authors = ["Kieran Leschinski <kieran.leschinski@unive.ac.at>"]
 maintainers = [
-    {name = "Kieran Leschinski", email="kieran.leschinski@unive.ac.at"},
-    {name = "Hugo Buddelmeijer", email="hugo@buddelmeijer.nl"},
+    "Kieran Leschinski <kieran.leschinski@unive.ac.at>",
+    "Hugo Buddelmeijer <hugo@buddelmeijer.nl>",
+    "Fabian Haberhauer <fabian.haberhauer@univie.ac.at>",
 ]
-classifiers=[
-    "Programming Language :: Python :: 3",
-    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
-    "Operating System :: OS Independent",
+readme = "README.md"
+repository = "https://github.com/AstarVienna/skycalc_ipy"
+documentation = "https://skycalc-ipy.readthedocs.io/en/latest/"
+classifiers = [
     "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
     "Topic :: Scientific/Engineering :: Astronomy",
 ]
-dependencies = [
-    # Try to keep the dependencies on lower versions that have a wheel
-    # package on PyPI, for minimumdependencies.yml
-    "numpy>=1.18.0",
-    "astropy>=4.0",
-    "httpx>=0.23",
-    "pyyaml>=5.3"
-]
 
-[project.optional-dependencies]
-test = [
-    "pytest>=5.0.0",
-    "pytest-cov",
-    # TODO: Should synphot be a normal dependency?
-    "synphot>=1.1.0",
-    "matplotlib>=3.2.0",
-]
-docs = [
-    "sphinx>=4.3.0",
-    "sphinx-rtd-theme>=0.5.1",
-    "jupyter_sphinx==0.2.3",
-    "sphinxcontrib-apidoc",
-    "nbsphinx",
-    "numpydoc",
-    "matplotlib>=3.2.0",
-]
+[tool.poetry.dependencies]
+python = "^3.8"
+numpy = "^1.24.4"
+astropy = "^5.2.2"
+httpx = "^0.23.0"
+pyyaml = "^6.0.0"
+
+synphot = { version = "^1.1.0", optional = true }
+
+[tool.poetry.group.test.dependencies]
```

Comment:
Maybe we should [configure RTD to install dependencies with poetry](https://docs.readthedocs.io/en/stable/build-customization.html#install-dependencies-with-poetry) instead of pip? If that works, then that's probably the easiest way forward.

Reply:
Now successfully builds from `fh/poetry` branch: https://skycalc-ipy.readthedocs.io/en/fh-poetry/